Resources/translations/SonataNewsBundle.de.xliff

```diff
@@ -443,6 +443,26 @@
                 <source>sonata.news.block.recent_posts</source>
                 <target>News - Letzte Artikel</target>
             </trans-unit>
+            <trans-unit id="form.label_number">
+                <source>form.label_number</source>
+                <target>Anzahl</target>
+            </trans-unit>
+            <trans-unit id="form.label_title">
+                <source>form.label_title</source>
+                <target>Titel</target>
+            </trans-unit>
+            <trans-unit id="form.label_mode">
+                <source>form.label_mode</source>
+                <target>Modus</target>
+            </trans-unit>
+            <trans-unit id="form.label_mode_public">
+                <source>form.label_mode_public</source>
+                <target>Öffentlich</target>
+            </trans-unit>
+            <trans-unit id="form.label_mode_admin">
+                <source>form.label_mode_admin</source>
+                <target>Admin</target>
```

Comment:
What does ``Admin`` mean?

Maybe something like ``Nur für Administratoren`` (``Only for Admins``) ??